common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,31 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  /**
+   * When memory usage of Netty is above this water mark, it's regarded as memory shortage.
```

Comment:
Sorry, you mean:
1. the change(`spark.network.netty.memCostWaterMark`) in this pr is ok
2. we merge `spark.executor.memory`(heap size) and `spark.memory.offHeap.size` (off-heap size, used by Tungsten) to be one in executor memory config.
Do I understand correctly ?